### DISCUSSION TITLE
hello if any one sees this does anyone know how to port hex to the  newst kade engine

### DISCUSSION THREAD

**Discussion by Michirugood** [KadeArchive/Kade-Engine#382] [Q&A] [answered]
## What is your suggestion, and why should it be implemented?
if anyone can help me get hex to the newest kade engine  please help me I want to have fun with the robo boi

**Comment by theDetourist** [comment, marked as answer]
If you just want the charts, replace them in your game files. If you want the whole thing, no way without the source code for it.
